columnflow/tasks/yields.py

```diff
@@ -57,6 +57,10 @@ class CreateYieldTable(
         description="string parameter to define the normalization of the yields; "
         "choices: '', per_process, per_category, all; empty default",
     )
+    suffix = luigi.Parameter(
```

Comment:
Can we name this `output_suffix`? I think such a suffix could be useful in many other tasks as well (e.g. plotting, but this should not be part of *this* PR of course), and since there are a couple other possible use cases for "suffixes", we could name this unambiguously right away.

Reply:
good point, it's changed now